MediaBrowser.Controller/BaseItemManager/BaseItemManager.cs

```diff
@@ -97,5 +102,28 @@ public bool IsImageFetcherEnabled(BaseItem baseItem, LibraryOptions libraryOptio
 
             return itemConfig == null || !itemConfig.DisabledImageFetchers.Contains(name, StringComparer.OrdinalIgnoreCase);
         }
+
+        /// <summary>
+        /// Creates the metadata refresh throttler.
+        /// </summary>
+        private void SetupMetadataThrottler()
+        {
+            MetadataRefreshThrottler = new Lazy<SemaphoreSlim>(() => new SemaphoreSlim(_metadataRefreshConcurrency));
```

Comment:
I've also been meaning to ask, why make it Lazy?

Reply:
I had this as a static field in an earlier commit. Since it needed to get a config value for the Semaphore creation, Lazy fit the scenario. Lazy would ensure the value is set to the same for all callers in the case of multiple threads calling it at once.
But now that it's refactored and non-static it doesn't have to be a Lazy. I fixed it